tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -326,6 +326,63 @@ test.describe( 'Nested Tabs tests', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Verify the correct working of the title alignment', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await setup( wpAdmin );
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' ),
+			tabsId = await editor.addWidget( 'nested-tabs', container );
+
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tabs-content .e-con.e-active' );
+
+		// Act.
+		// Add Icons.
+		await setIconsToTabs( page, TabsIcons );
+		const activeTab = editor.getPreviewFrame().locator( '.e-normal.e-active' );
+
+		// Tabs styling scenario 1: Direction: Top, Align Title: Left, Icon Position: Right.
```

Comment:
I loved the new associative styling of the test,
More readable.

Reply:
Nice.